src/model/migrations.ts

```diff
@@ -599,6 +607,18 @@ export default async function runMigrations() {
 
   migrations.push(v15);
 
+  /*
+   *************** Migration v16 ******************
+   * clean mmkv ration / dominant color
+   * https://linear.app/rainbow/issue/RNBW-2885
+   */
+  const v16 = async () => {
+    storageAspectRatio.clearAll();
+    storageDominantColor.clearAll();
+  };
+
+  migrations.push(v16);
+
```

Comment:
So the migration v16 you did recently was reversed and it wasn't released, right?